portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase

```diff
@@ -0,0 +1,76 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Check that is possible to delete a blog"
+	@priority = 4
+	test CanDeleteBlog {
+		task ("And 2 Blog Entries exists") {
+			JSONBlog.addEntry(
+				entryContent = "Blog Entry Content 1",
+				entryTitle = "Blog Entry Title 1");
+
+			JSONBlog.addEntry(
+				entryContent = "Blog Entry Content 2",
+				entryTitle = "Blog Entry Title 2");
+
+			Refresh();
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blog Entry Content 1",
+				entryTitle = "Blog Entry Title 1");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blog Entry Content 2",
+				entryTitle = "Blog Entry Title 2");
+		}
+
+		task ("When the User deletes Blog Entry 1") {
+			BlogsEntry.moveToRecycleBinPG(
+				entryContent = "Blog Entry Content 1",
+				entryTitle = "Blog Entry Title 1");
+		}
+
+		task ("Then Blog Entry 1 should not exist") {
+			BlogsEntry.viewNoPG(entryTitle = "Blog Entry Title 1");
+		}
+
+		task ("And Blog Entry 2 should exist") {
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blog Entry Content 2",
+				entryTitle = "Blog Entry Title 2");
+		}
+	}
+
+}
```

Comment:
Hi, remember our chat about variables? Can you create two variables one of the blog entry 1 and one for blog entry 2?

Reply:
Okay, I'll do it. Thanks for the review!